elastic/github-commands/action.yml

```diff
@@ -20,14 +20,21 @@ runs:
       with:
         github-token: ${{ inputs.github-token }}
         script: |
+          let commands = '- `run` `docs-build` : Re-trigger the docs validation. (use unformatted text in the comment!)'
+          // See https://github.com/elastic/logstash/pull/18383
+          if ((context.repo.repo === 'logstash' || context.repo.repo === 'oblt-actions') && context.repo.owner === 'elastic') {
+            commands += `
+            - \`/run exhaustive tests\` : Run the exhaustive tests Buildkite pipeline.`
+          }
+
           const body = `
             ### :robot: GitHub comments
 
             <details><summary>Expand to view the GitHub comments</summary>
```

Comment:
I fell we should somehow get rid of the collapsive and show the commands as they are